app/src/main/java/com/cyberbot/checkers/game/GameEnd.java

```diff
@@ -0,0 +1,21 @@
+package com.cyberbot.checkers.game;
+
+import org.jetbrains.annotations.NotNull;
+
+public class GameEnd {
+    private final PlayerNum winner;
+    private final String reason;
```

Comment:
The win reason has to be an changed to an enum to allow for localization.

Reply:
Ok, will be fixed in a second.